features/support/saucelabs/browsers.rb

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is this file necessary? Shouldn't the `browser` method in the `SauceDriver` module be enough?